sdk/packages/eternum/src/provider/index.ts

```
@@ -13,18 +27,39 @@
   return contract.address;
 };
 
+/**
+ * Higher order function that adds event emitter functionality to a class
+ *
+ * @param Base - The base class to extend
+ * @returns A new class with event emitter capabilities
+ */
 function ApplyEventEmitter<T extends new (...args: any[]) => {}>(Base: T) {
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)